department-of-veterans-affairs/caseflow
# Ki/APPEALS-35254 - Updates to All Steps of the Intake form

Resolves [Updates to All Steps of the Intake form](https://jira.devops.va.gov/browse/APPEALS-35254)

# Description
This PR Updates to all 3 Steps of the Correspondence Intake Form to match the wireframes and CSS requirements.

## Acceptance Criteria

## Testing Plan

1. Go to the Intake Form of a Correspondence
2. Verify that the styling matches Step 1, Step 2, and Step 3 with options selected.
3. Verify that the styling matches during Step 3 if NO options are selected at all.
    1. Also verify that the new "Correspondence is not related..." populates correctly when no options are selected.

# Frontend
## User Facing Changes

### Updated Background Colors
![Screenshot 2024-01-05 at 3 07 10 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/850a7cfc-b59a-43e6-891f-558aea4f6b75)
![Screenshot 2024-01-05 at 3 07 17 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/5c337da6-0652-4c83-9d18-6c43b9cdf12d)

### Updated Spacing
![Screenshot 2024-01-05 at 3 28 26 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/bebf3dff-250e-40fc-b773-ca6d222e8921)
![Screenshot 2024-01-05 at 3 28 42 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/0d65ff71-d455-4b9e-acf7-b715c8700f21)
<img width="783" alt="Screenshot 2024-01-05 at 3 49 11 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/232119c1-a0fc-43a8-8068-9d62574e9a58">
<img width="621" alt="Screenshot 2024-01-05 at 3 55 42 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/ae4a3311-7380-483a-a1ad-b4448238699e">

### Updated Alignment for Tasks related
![Screenshot 2024-01-05 at 3 32 12 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/ac0c4c72-81c8-4f8b-99c9-acbc99e9c467)

### Horizontal Rule for AC 4 between Task options on Step 2
![Screenshot 2024-01-08 at 2 48 35 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/247b713d-383c-4811-b472-3f18cad896c9)

### Nothing Selected "Correspondence is not related..."
![Screenshot 2024-01-08 at 2 43 09 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/45ccc3c0-c595-45d1-aea5-09b00aa5c23a)
